<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Startup</class>
 <widget class="QDialog" name="Startup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HikLoad - Startup Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
     <item row="1" column="2">
      <widget class="QLineEdit" name="username"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Downloads folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server IP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="server_ip"/>
     </item>
     <item row="4" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="downloads_folder">
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downloads_folder_button">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>318</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Folder behavior</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="folder_behavior">
            <property name="currentText">
             <string>Do not create any more folders</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <item>
             <property name="text">
              <string>Do not create any more folders</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Create one folder per camera</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Create one folder per day</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Create one folder per month</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Create one folder per year</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Download Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="download_type">
            <property name="currentText">
             <string>Videos</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <item>
             <property name="text">
              <string>Videos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Photos (EXPERIMENTAL!)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Video format</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="video_format">
            <property name="currentText">
             <string>mkv</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <item>
             <property name="text">
              <string>mkv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mp4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>avi</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Start date</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateTimeEdit" name="start_date">
            <property name="date">
             <date>
              <year>2021</year>
              <month>12</month>
              <day>30</day>
             </date>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="timeSpec">
             <enum>Qt::UTC</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>End date</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateTimeEdit" name="end_date">
            <property name="date">
             <date>
              <year>2022</year>
              <month>12</month>
              <day>29</day>
             </date>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="timeSpec">
             <enum>Qt::UTC</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="debug">
           <property name="text">
            <string>Enable debug mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="force">
           <property name="text">
            <string>Force downloading and saving of files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="localtime">
           <property name="text">
            <string>Save filenames using date in local time instead of UTC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ffmpeg">
           <property name="text">
            <string>Enable ffmpeg and disable downloading directly from server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="forcetranscoding">
           <property name="text">
            <string>Force transcoding if downloading directly from server</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="cameras">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>Cameras from which to download</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Test the connection to see the channels</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>318</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="test_connection_button">
       <property name="text">
        <string>Test connection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_downloading_button">
       <property name="text">
        <string>Start downloading</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
